tox.ini

```diff
@@ -49,5 +52,13 @@ deps =
     sphinxdev: https://github.com/sphinx-doc/sphinx/archive/refs/heads/master.zip
 commands =
     pytest {posargs} tests/
+    # This is a short-term hack to allow us to have a Python 3-specific syntax
+    # and swap it out for a Python 2.7 version while we're building.
+    # This terrible solution is only acceptable because we will remove it again very soon
+    # See: https://github.com/readthedocs/sphinx_rtd_theme/pull/1369
+    py27: cp docs/demo/test_py_module/test.py docs/demo/test_py_module/test_py3.py
+    py27: cp docs/demo/test_py_module/test_py27.py docs/demo/test_py_module/test.py
     !html4: sphinx-build -b html -Dhtml4_writer=0 -d {envtmpdir}/doctrees docs/ {envtmpdir}/html
     html4: sphinx-build -b html -Dhtml4_writer=1 -d {envtmpdir}/doctrees docs/ {envtmpdir}/html
+    py27: cp docs/demo/test_py_module/test_py3.py docs/demo/test_py_module/test.py
+    py27: rm docs/demo/test_py_module/test_py3.py
```

Comment:
@agjohnson I haven't had a proper look as to why this didn't break before now. I don't understand how our test builds got away with having a module that has invalid Python 2.7 syntax. The issue didn't come up until here.

The work-around is ugly. I created a Python 2.7 version of the same module, and during tests, I swap that python file with the Python 3 version.

It's ugly, but it works, and we can remove it very soon again.

Reply:
Mm yeah that's quite fiddly and hidden. Any reason not to use conditional reST instead?

I've pushed up an example at https://github.com/readthedocs/sphinx_rtd_theme/compare/agj/pr-1369 and will see what CI says.

Reply:
27 tests are okay on my branch, and output looks correct:

![image](https://user-images.githubusercontent.com/1140183/200049786-a0336162-2d72-416b-ab97-fba5c23ecb10.png)

Reply:
And all tests are green and output seems to look okay. I'm going to proceed with bringing in my branch here and releasing